test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -94,17 +119,21 @@ const setUpResponse = (filePath) => {
   return mock;
 };
 
-async function setup(t, locale) {
+async function setup(t, locale, fileName) {
+  const options = mocksWithInteractionCodeFlow.includes(fileName) ? optionsForInteractionCodeFlow : {};
   const widgetView = new PageObject(t);
-  await widgetView.navigateToPage();
+  await widgetView.navigateToPage({ render: false });
+  await widgetView.mockCrypto();
+  await t.setNativeDialogHandler(() => true);
```

Comment:
Do we need to `mockCrypto` and `setNativeDialogHandler` for mocks that are now with `InteractionCodeFlow` ?

Reply:
No

Reply:
Is it possible to only call mockCrypto and setNativeDialogHandler for the mocks that need it ?

Reply:
Done